src/record/import/utils/file.ts

```diff
@@ -15,11 +15,17 @@ export const readFile: (
   if (format === "json" && encoding !== "utf8") {
     throw new Error("source file is JSON and JSON MUST be encoded with UTF-8");
   }
-  const stream = fs
-    .createReadStream(filePath)
-    .pipe(iconv.decodeStream(encoding));
-  const content = await readStream(stream);
-  return { content, format };
+
+  return new Promise((resolve, reject) => {
```

Comment:
The Promise constructor accept an async function, so you can simplify like this:

```typescript
new Promise<string>(async (resolve, reject) => {
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/Promise

Reply:
@ueokande 

There is an ESLint error if we use async along with Promise. So, I used self-invoking function.

```
ESLint: Promise executor functions should not be async.(no-async-promise-executor)
```

https://github.com/cybozu/eslint-config/blob/4016bf4f299867cd13cf4c53c9b4732446b16588/lib/base.js#L60
https://eslint.org/docs/latest/rules/no-async-promise-executor

Reply:
I got it, thanks for your information.  I learned a lot.